infra/build/Makefile

```diff
@@ -26,14 +28,23 @@ BATS_ASSERT_VERSION := 2.0.0
 BATS_MOCK_VERSION := 1.0.1
 PACKER_VERSION := 1.4.4
 TERRAGRUNT_VERSION := 0.25.3
-KUSTOMIZE_VERSION := 3.6.0
+KUSTOMIZE_VERSION := 3.6.1
+# Updated by Update Tooling Workflow
 # Hold KPT at 1.0.0-beta.15 till resolved: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1190
 KPT_VERSION := 1.0.0-beta.15
+# Updated by Update Tooling Workflow
 CFT_CLI_VERSION := 0.4.5
-ASMCLI_VERSION := 1.12
+# Updated by Update Tooling Workflow
 KUBECTL_VERSION := 1.23.13
 ALPINE_VERSION := 3.16
 
+# For developer-tools-krm
+GOLANGCI_VERSION := 1.40.1
+ASMCLI_VERSION := 1.12
+# Updated by Update Tooling Workflow
+GATOR_VERSION := 3.9.2
+OPA_VERSION := 0.42.2
+
 DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.8.1
```

Comment:
```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.8.2
```

Reply:
Thanks, yeah, thought about that after the fact. :)